src/sentry/models/rule.py

```diff
@@ -130,3 +131,19 @@ class RuleActivity(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_ruleactivity"
+
+
+def in_fourteen_days():
```

Comment:
Maybe leave a comment that this is used in migrations and can't be removed/moved.

Reply:
@markstory if for some reason we need to change it, we edit the function itself right? It'd just be weird to have `in_fourteen_days` set 30 days (or some other number)?

Reply:
Thought about it more and I'm just gonna remove it - I was doing it to make it a teeny bit easier when populating to not have to find 14 days in the future but it's not a big deal.